### PR TITLE
Added a directive to fix some minor bugs

### PR DESCRIPTION
**Pull request by ChisatoNishikigi73**
## Description
%This is a PR based on #1729, after pulling the new version%

1.Generate GM Handbook into a folder
2.Add the "rotation" command to get rotation information (useful for resource writing).
3.Fixed scene ID and number of monsters in "Killall" command
## Issues fixed by this PR

issue#1686

- [x] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by 4Benj** [comment]
Why was the original closed?

**Comment by Birdulon** [comment]
You don't have to close your PR when you make changes to it, even with force pushes, by the way.

**Comment by jie65535** [comment]
Maybe just don't want to keep the original messy commit log.

**Comment by ChisatoNishikigi73** [comment]
> Why was the original closed?



> You don't have to close your PR when you make changes to it, even with force pushes, by the way.

Since I couldn't resolve the merge conflict problem, I had to delete the previous commit and write again

**Comment by Birdulon** [comment]
> Since I couldn't resolve the merge conflict problem, I had to delete the previous commit and write again

If you force push to the same branch, it will still work without needing a new PR

**Comment by ChisatoNishikigi73** [comment]
> If you force push to the same branch, it will still work without needing a new PR

I see. Thank you！

**Comment by ChisatoNishikigi73** [comment]
Can this PR merge be done? I have another PR about respawnTeam

File: src/main/resources/languages/ja-JP.json [hunk 1]
```diff
@@ -207,8 +207,8 @@
       "description": "ユーザーの権限を付与または削除します"
     },
     "position": {
-      "success": "座標: %s、%s、%s\nシーンID: %s",
-      "description": "座標を取得する"
+      "success": "座標: %s、%s、%s\n回転：%s, %s, %s\nシーンID: %s",
```
Comment (Birdulon) [inline src/main/resources/languages/ja-JP.json]:
```suggestion
      "success": "座標：%s、%s、%s\n回転：%s、%s、%s\nシーンID: %s",
```

**Review by Birdulon — APPROVED** [review]

**Pull request merged**